dl_binaries.ps1

```diff
@@ -31,7 +37,7 @@ $targets | foreach-object {
     if (!(test-path -path "binaries/$version/$target")) { New-Item -Path "binaries/$version/$target" -ItemType Directory -force | out-null }
 
     Write-Output "downloading $path"    
-    invoke-webrequest -uri "https://update.tabnine.com/bundles/$path/TabNine.zip" -outfile "binaries/$path/TabNine.zip"
+    invoke-webrequest -uri "$TABNINE_UPDATE_SERVICE/bundles/$path/TabNine.zip" -outfile "binaries/$path/TabNine.zip"
     # Stop this iteration if the download failed
     if ($LastExitCode -eq 0) {return}
```

Comment:
this needs to be `-gt` no?

Reply:
It needs to be `-ne`, but this is already handled by #81, so this doesn't need to change it a second time. If you look at [master](https://github.com/codota/tabnine-nvim/blob/master/dl_binaries.ps1#L42), it includes `-ne` rather than `-eq`.